crt_portal/cts_forms/question_group.py

```diff
@@ -14,8 +25,23 @@ def __init__(self, form, fields, group_name='', help_text='', ally_id='', option
         self.ally_id = ally_id
         self.label_cls = label_cls
         self.help_cls = help_cls
+        self.extra_validation_fields = extra_validation_fields
 
     def __iter__(self):
         for name in self.fields:
             field = self.form.fields[name]
             yield BoundField(self.form, field, name)
+
+    def errors(self):
```

Comment:
I really like this approach! It gives actionable error output without being overwhelming and the logic is pretty intuitive.